mod-source-record-manager/src/main/resources/folijet/mod-source-record-manager/features/job-execution.feature

```diff
@@ -5,35 +5,278 @@ Feature: Source-Record-Manager
+            "record": "00182cx  a22000851  4500001000900000004000800009005001700017008003300034852002900067\u001e10245123\u001e9928371\u001e20170607135730.0\u001e1706072u    8   4001uu   0901128\u001e0 \u001fbfine\u001fhN7433.3\u001fi.B87 2014\u001e\u001d",
+            "order": 6
+          }
+        ],
+        "recordsMetadata": {
+          "last": true,
+          "counter": 7,
+          "contentType": "MARC_RAW"
+        }
+      }
+    """
+
+    * def arrayOfJobExecutorsFilteredBySubordinationType =
+    """
+      function (subordinationType) {
+        return response.jobExecutions.filter(jobExecution => jobExecution.subordinationType === subordinationType && isJobExecutionValid(jobExecution));
+      }
+    """
+
+    * def isJobExecutionValid =
+    """
+      function(jobExecution) {
+        return jobExecution != null
+          && jobExecution.id != null
+          && jobExecution.parentJobId != null
+      }
+    """
+  @Positive
   Scenario: Test init job execution with 1 file
     * print 'Test init job execution with 1 file'
 
-  @Undefined
+    Given path 'change-manager', 'jobExecutions'
+    And request oneFileJobExecution
+    When method POST
+    Then status 201
+    And match response.parentJobExecutionId != null
+
+    * def parentJobExecution = response.jobExecutions[0]
+    * def isJobExecutionValid = call isJobExecutionValid parentJobExecution
+    And match parentJobExecution.sourcePath != null
+    And match parentJobExecution.fileName != null
+    And match isJobExecutionValid == true
+    And match parentJobExecution.subordinationType == 'PARENT_SINGLE'
+    And match parentJobExecution.status == 'NEW'
+    And match parentJobExecution.id == parentJobExecution.parentJobId
+
+  @Positive
   Scenario: Test init job execution with multiple files
     * print 'Test init job execution with multiple files'
 
-  @Undefined
+    * def expectedJobExecutionsNumber = expectedParentJobExecutions + expectedChildJobExecutions;
+
+    Given path 'change-manager', 'jobExecutions'
+    And request multipleFileJobExecution
+    When method POST
+    Then status 201
+    And match response.parentJobExecutionId != null
+    And assert response.jobExecutions.length == 3
+
+    * def numberOfParent = call arrayOfJobExecutorsFilteredBySubordinationType 'PARENT_MULTIPLE'
+    And assert numberOfParent.length == expectedParentJobExecutions
+
+    * def numberOfChild = call arrayOfJobExecutorsFilteredBySubordinationType 'CHILD'
+    And assert numberOfChild.length == expectedChildJobExecutions
+
+  @Positive
   Scenario: Test return of children job executions for multiple files
     * print 'Init job execution with multiple files, get children of that job execution'
 
-  @Undefined
+    Given path 'change-manager', 'jobExecutions'
+    And request multipleFileJobExecution
+    When method POST
+    Then status 201
+
+    * def parentId = response.jobExecutions[0].id
+
+    Given path 'change-manager', 'jobExecutions', parentId
```

Comment:
hello @RomanChernetskyi, looks like here we need to use `/change-manager/jobExecutions/{parentJobId}/children` endpoint to get children of the multi-file job execution